### ISSUE TITLE
The GroupCreate class has never been used

### ISSUE THREAD

**Issue by dark-wind** [adamwojs/php-activiti-api#2]
The GroupCreate class has never been used.
createGroup only used a array.

**Comment by adamwojs** [comment]
You are right, thank you for reporting. `Activiti\Client\Model\Group\GroupCreate` should be argument for `GroupServiceInterface::createGroup` method. Feel free to open PR :wink:

**Comment by dark-wind** [comment]
Thank you for your reply.
i will fix it recently.
But i am buzyying now , i plan do it next week
